Configuration/HLT/python/autoCondHLT.py

```diff
@@ -41,27 +41,27 @@
-    'run2_hlt_Fake1'         : ('run2_hlt_relval'         ,l1Menus['Fake1']),
-    'run2_hlt_Fake2'         : ('run2_hlt_relval'         ,l1Menus['Fake2']),
-    'run3_hlt_FULL'          : ('run3_hlt_relval'         ,l1Menus['FULL']),
-    'run3_hlt_GRun'          : ('run3_hlt_relval'         ,l1Menus['GRun']),
-    'run3_hlt_2022v15'       : ('run3_hlt_relval'         ,l1Menus['2022v15']),
-    'run3_hlt_HIon'          : ('run3_hlt_relval'         ,l1Menus['HIon']),
-    'run3_hlt_PIon'          : ('run3_hlt_relval'         ,l1Menus['PIon']),
-    'run3_hlt_PRef'          : ('run3_hlt_relval'         ,l1Menus['PRef']),
+    'run1_hlt_Fake'          : ('run2_hlt'                ,l1Menus['Fake']),
+    'run2_hlt_Fake'          : ('run2_hlt'                ,l1Menus['Fake']),
+    'run2_hlt_Fake1'         : ('run2_hlt'                ,l1Menus['Fake1']),
+    'run2_hlt_Fake2'         : ('run2_hlt'                ,l1Menus['Fake2']),
+    'run3_hlt_FULL'          : ('run3_hlt'                ,l1Menus['FULL']),
+    'run3_hlt_GRun'          : ('run3_hlt'                ,l1Menus['GRun']),
+    'run3_hlt_2022v15'       : ('run3_hlt'                ,l1Menus['2022v15']),
+    'run3_hlt_HIon'          : ('run3_hlt'                ,l1Menus['HIon']),
+    'run3_hlt_PIon'          : ('run3_hlt'                ,l1Menus['PIon']),
+    'run3_hlt_PRef'          : ('run3_hlt'                ,l1Menus['PRef']),
 
-    'run1_data_Fake'         : ('run1_data'               ,l1Menus['Fake']),
-    'run2_data_Fake'         : ('run2_data_relval'        ,l1Menus['Fake']),
-    'run2_data_Fake1'        : ('run2_data_relval'        ,l1Menus['Fake1']),
-    'run2_data_Fake2'        : ('run2_data_relval'        ,l1Menus['Fake2']),
-    'run3_data_FULL'         : ('run3_data_relval'        ,l1Menus['FULL']),
-    'run3_data_GRun'         : ('run3_data_relval'        ,l1Menus['GRun']),
-    'run3_data_2022v15'      : ('run3_data_relval'        ,l1Menus['2022v15']),
-    'run3_data_HIon'         : ('run3_data_relval'        ,l1Menus['HIon']),
-    'run3_data_PIon'         : ('run3_data_relval'        ,l1Menus['PIon']),
-    'run3_data_PRef'         : ('run3_data_relval'        ,l1Menus['PRef']),
+    'run1_data_Fake'         : ('run2_data'               ,l1Menus['Fake']),
+    'run2_data_Fake'         : ('run2_data'               ,l1Menus['Fake']),
+    'run2_data_Fake1'        : ('run2_data'               ,l1Menus['Fake1']),
+    'run2_data_Fake2'        : ('run2_data'               ,l1Menus['Fake2']),
+    'run3_data_FULL'         : ('run3_data_prompt'        ,l1Menus['FULL']),
+    'run3_data_GRun'         : ('run3_data_prompt'        ,l1Menus['GRun']),
+    'run3_data_2022v15'      : ('run3_data_prompt'        ,l1Menus['2022v15']),
+    'run3_data_HIon'         : ('run3_data_prompt'        ,l1Menus['HIon']),
+    'run3_data_PIon'         : ('run3_data_prompt'        ,l1Menus['PIon']),
+    'run3_data_PRef'         : ('run3_data_prompt'        ,l1Menus['PRef']),
```

Comment:
Why is this change (`run3_data` -> `run3_data_prompt`) useful here ?

GTs like `run3_data_GRun` are mainly used in HLT tests that run HLT+RECO (so, HLT and offline reconstruction in the same job; afaik, no production wf runs HLT+RECO in one go, but it is a functionality that HLT tries to maintain). I don't know if there is a realistic scenario where we'd run HLT+RECO on real data: if there is one, what would the most appropriate GT be (`run3_data` or `run3_data_prompt`)?

(If the answer is not obvious, I suggest to leave `run3_data` and HLT can take care of changing this later with a backport if needed.)

Reply:
`run3_data` points to the ReRECO GT, so should be used only once we cut a rereco GT. When we do that it will have the snapshot time, and that will not change for another year or so, on the other hand the prompt GT we resnapshot more often (like every month or so). I think using the `run3_data_prompt` is the appropriate one for HLT tests too, the `run3_data` would use the last visible conditions only (i.e. if you run this on data from the upcoming MWGR, it will use conditions from December 2022, since that's when we made the rereco GT)